pacai/bin/capture.py

```diff
@@ -610,8 +608,9 @@ def readCommand(argv):
     args['redTeamName'] = options.red
     args['blueTeamName'] = options.blue
 
-    if options.fixRandomSeed:
-        random.seed(FIXED_SEED)
+    # If no seed entry generate a random seed value
+    seed = random.seed(options.seed) if options.seed else random.randint(0, 2**32)
```

Comment:
Never have an if on a single line.
This is in PEP-8, but flake doesn't enforce it (so it is easy to miss).
The key here is that we always want our code to be clear, even at first glance.
It is easy to miss compound statements like this.